webapp/login/useResetPasswordFormState.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+
```

Comment:
should we move this file, loginForm, passwordValidator, acceptInvitationForm into an inner folder called components?